tests/testthat/test-problems.R

```diff
@@ -11,16 +11,15 @@ test_that("problems with data parsing works for single files", {
 })
 
 test_that("problems works for multiple files", {
-
   out1 <- file.path(tempdir(), "out1.txt")
   out2 <- file.path(tempdir(), "out2.txt")
   on.exit(unlink(c(out1, out2)))
 
   writeLines("x,y\n1,2\n1,1.x\n2,2", out1)
   writeLines("x,y\n3.x,4\n1,2\n2,2", out2)
 
-  expect_warning(
-    x <- vroom(c(out1, out2), delim = ",", col_types = "dd", altrep=F)
+  expect_snapshot(
+    x <- vroom(c(out1, out2), delim = ",", col_types = "dd", altrep = F)
```

Comment:
```suggestion
    x <- vroom(c(out1, out2), delim = ",", col_types = "dd", altrep = FALSE)
```

Seeing this makes me this we should probably subject the entire file to styler, since we're touching it.